src/main/resources/test.sql

```diff
@@ -0,0 +1,16 @@
+single domain
+v1/categories/{id}?status=ACTIVE  GET(specific client)
```

Comment:
[Optional]: status=ACTIVE ehtiyac yoxdur, onu database seviyyesinde edeceksiniz

Reply:
I deleted this. Thanks